extras/help_generator/writers.py

```diff
@@ -184,8 +187,7 @@ def write_helpindex(helpdir):
         html_list.append('</tr></table></center>')
         html_list.append('<center><table class="commands" id="%s">'
                          % doubles[0])
-        keyfunc = lambda x: x.lower().rsplit('/', 1)[1]
-        for item in sorted(filelist, key=keyfunc):
+        for item in sorted(filelist, key=key_func):
```

Comment:
There's no technical reason. I find find these multi-line expressions ugly and try to avoid them whenever possible.